docs/plugins/testing_workshop_docs/Readers-and-fixtures.md

```diff
@@ -0,0 +1,147 @@
+This article will be more valuable if you are familiar with the Python programming language and the Napari software. It is the third in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 15:42. This article should stand on its own and is a summary of the information in the video. The other articles are:  
+* Article 1: [Python’s assert keyword](./Pythons-assert-keyword.md)  
+* Article 2: [Pytest testing framework](./Pytest-testing-frameworks)  
+* Article 3: This article  
+* Article 4: [Test coverage](./Test-coverage)   
+* Article 5: Testing widgets  
+
+This article covers:   
+* [Readers](#reader)  
+* [Built-in fixtures](#built-in-fixtures)  
+* [Custom fixtures and round-trip tests](#custom-fixtures-and-round-trip-tests)  
+* [Enclosed testing](#enclosed-testing)  
+  
+## Resources  
```

Comment:
For consistency, I suggest

```suggestion
**This article covers:**
* [Readers](#reader)  
* [Built-in fixtures](#built-in-fixtures)  
* [Custom fixtures and round-trip tests](#custom-fixtures-and-round-trip-tests)  
* [Enclosed testing](#enclosed-testing)  

**Resources**
```

Reply:
I inserted a ## heading and a summary sentence above these 2 in each article. These are still ### headings, as they should be following a ## heading.